src/client/object/ObjectFeed/ObjectFeed.js

```diff
@@ -65,9 +65,15 @@ export default class ObjectFeed extends React.Component {
   state = {
     loadingPropositions: false,
   };
+  // eslint-disable-next-line react/sort-comp,consistent-return
+  getNewsPermlink = () => {
+    if (!isEmpty(this.props.match.params[1]) || !isNil(this.props.match.params[1])) {
+      return this.props.wobject?.newsFeed?.permlink;
+    }
+  };
 
```

Comment:
getNewsPermlink = () => {
 if (isEmpty(this.props.match.params[1]) || isNil(this.props.match.params[1])) return null;

      return this.props.wobject?.newsFeed?.permlink;
  };